sql/moz-fx-data-shared-prod/search/urlbar_clients_daily/temp_urlbar_clients_daily.sql

```diff
@@ -0,0 +1,48 @@
+  -- daily aggregates
+SELECT
+  client_id,
+  DATE(submission_timestamp) AS submission_date,
+  --  urlbar searchmode (needs to be added to clients_daily)
+  `mozfun.map.sum`(ARRAY_CONCAT_AGG(payload.processes.parent.keyed_scalars.urlbar_searchmode_bookmarkmenu)) AS searchmode_bookmarkmenu,
```

Comment:
```suggestion
  mozfun.map.sum(ARRAY_CONCAT_AGG(payload.processes.parent.keyed_scalars.urlbar_searchmode_bookmarkmenu)) AS searchmode_bookmarkmenu,
```

Backticks are only necessary when referencing a project that contains `-` which is otherwise illegal in BigQuery SQL. Part of the motivation for `mozfun` is that it's a simple name that can be referenced without backticks.

Reply:
cool, thanks. that's good to know.